lib/just_audio_web.dart

```diff
@@ -154,7 +154,8 @@ class Html5AudioPlayer extends JustAudioPlayer {
     _audioElement.src = url;
     _audioElement.preload = 'auto';
     _audioElement.load();
-    final duration = await _durationCompleter.future;
+    var duration = await _durationCompleter.future;
+    duration = duration != double.infinity ? duration : 0;
```

Comment:
When the duration is unknown, return -1 to have it correctly handled on the Flutter side.